<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QTextBrowser" name="console_output"/>
            </item>
            <item>
             <widget class="QTextBrowser" name="console_output2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Layer Number</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="layerNumber">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>200</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Feature Map Number</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="featureMapNumber">
              <property name="maximum">
               <number>600</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="layerButton">
            <property name="text">
             <string>Get Layer Configuration</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="outputFeatureMaps">
            <property name="text">
             <string>Show Output Feature Maps</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="specificFeatureMap">
            <property name="text">
             <string>Specific feature map</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="displayOutput">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="loadModel">
        <property name="text">
         <string>Load Model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="modelSummary">
        <property name="text">
         <string>MOdel Summary</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="inputImage">
        <property name="text">
         <string>Laod Input Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="computeOutput">
        <property name="text">
         <string>Predict Output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="displayInput">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="displayPreprocessedInput">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>958</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionDuplicate_window"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDuplicate_window">
   <property name="text">
    <string>Duplicate window</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
